tests/commands/run_test.py

```diff
@@ -14,11 +14,12 @@
 from pre_commit.commands.run import _has_unmerged_paths
 from pre_commit.commands.run import get_changed_files
 from pre_commit.commands.run import run
+from pre_commit.ordereddict import OrderedDict
 from pre_commit.runner import Runner
 from pre_commit.util import cmd_output
 from pre_commit.util import cwd
 from testing.auto_namedtuple import auto_namedtuple
-from testing.fixtures import make_consuming_repo
+from testing.fixtures import add_config_to_repo, make_consuming_repo
```

Comment:
weird, there's a pre-commit hook that should split this line, does this line change when `pre-commit run --all-files` ? (might be a side-effect of rebase?)